codigo/src/views/JuegoView.java

```diff
@@ -0,0 +1,75 @@
+package views;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+import negocio.Jugador;
+import negocio.Pareja;
+import negocio.Partida;
+
+public class JuegoView {
```

Comment:
implements Serializable and generate serial. DTO instead of view. Same but add setters..